securedrop_client/logic.py

```diff
@@ -485,7 +485,7 @@ def update_sources(self):
         """
         sources = list(storage.get_local_sources(self.session))
         if sources:
-            sources.sort(key=lambda x: x.last_updated, reverse=True)
+            sources.sort(key=lambda x: x.last_updated)
```

Comment:
with this change doesn't it mean that for _existing_ `SourceWidget` items, when the list is updated, the last updated items don't float to the top? For example this situation:

1. Source A is added. It's at the top.
2. Source B is added. It's now at the top with Source A below.
3. A new message appears from Source A. 

I would expect source A to be at the top, but it's still source B

Reply:
yes, logically that's what should be happening, you're right. i wonder why i didn't see that when inserting 2 new sources - will test again and figure out why that does happen OR if i see that happening will change the list to return in descending order again

Reply:
Turns out what I have in this PR is correct, because year-mon-day hour:min:sec gets sorted so that oldest is on the top. If you print out `sources` which is sorted like `sources.sort(key=lambda x: x.last_updated)` you'll see:

`[<Source new>, <Source newer>, <Source newest>]`

and if you run `select journalist_designation,last_updated from sources; ` on `svs.sqlite` you'll see:

```
new|2020-03-10 20:21:52.010863
newer|2020-03-10 20:22:05.525741
newest|2020-03-10 20:22:09.956040
```

Reply:
quick status on this PR: after re-review my other comments are addressed! thanks for the rapid changes. but tomorrow AM I want to double check the ordering behavior (first implemented in #158) because I'm not seeing the most recently updated (by the source) items float to the top... I'm not sure if it's introduced here (we may have been relying on the redrawing of the source list for the reordering) or is an issue on `master`

Reply:
before we were using `addItem` which appends to the bottom of the list. this is why the client used to sort the sourcelist in reverse order

Reply:
I agree that new sources are added to the top, I was wondering when a source conversation is updated for a source widget that is already drawn, why we don't see the source widget move to the top of the list. But: it appears that this is an existing issue on `master`, so I've opened https://github.com/freedomofpress/securedrop-client/issues/914 to track that